src/compiler/scala/tools/nsc/javac/JavaScanners.scala

```diff
@@ -702,6 +716,108 @@ trait JavaScanners extends ast.parser.ScannersCommon {
         in.next()
       }
 
+    /** read a triple-quote delimited text block, starting after the first three
+      * double quotes
+      */
+    private def getTextBlock(): Unit = {
+      // Open delimiter is followed by optional space, then a newline
+      while (isWhitespace(in.ch) && in.ch != CR && in.ch != LF && in.ch != SU) {
```

Comment:
maybe avoid using `isWhitespace` since it's not really doing what we want here?

Acording to https://docs.oracle.com/javase/specs/jls/se16/html/jls-3.html#jls-3.10.6:

```suggestion
      while (in.ch = ' ' || in.ch == '\t' || in.ch == FF) {
```

Reply:
It is shorter and simpler. 👍